packages/@lwc/integration-karma/test/component/LightningElement.attachInternals/elementInternals/formAssociated/index.spec.js

```diff
@@ -0,0 +1,88 @@
+import { createElement } from 'lwc';
+
+import NotFormAssociated from 'x/notFormAssociated';
+import FormAssociated from 'x/formAssociated';
+
+if (typeof ElementInternals !== 'undefined') {
+    // Verify ElementInternals proxy getter throws error.
+    it('form-related operations and attributes should throw DOMException for non-form-associated custom elements.', () => {
```

Comment:
Added these sanity tests just to confirm that form associated behavior works as expected with the proxy.